packages/client-core/src/components/World/LoadLocationScene.tsx

```diff
@@ -88,11 +87,21 @@ export const useLoadLocation = (props: { locationName: string }) => {
 }
 
 export const useLoadScene = (props: { projectName: string; sceneName: string }) => {
+  const key = `projects/${props.projectName}/${props.sceneName}`
+  const resourceQuery = useFind(staticResourcePath, {
+    query: {
+      key
+    }
+  })
+
   useEffect(() => {
-    if (!props.sceneName || !props.projectName) return
-    const key = `${props.projectName}/${props.sceneName}`
-    const url = getState(DomainConfigState).cloudDomain + `/projects/${key}`
-    getMutableState(LocationState).currentLocation.location.sceneId.set(key)
-    return GLTFAssetState.loadScene(url, key)
-  }, [])
+    if (!resourceQuery.data.length) return
+    const resource = resourceQuery.data[0]
+    getMutableState(LocationState).currentLocation.location.sceneId.set(resource.id)
+    const unload = GLTFAssetState.loadScene(resource.url, resource.id)
```

Comment:
Changing this to use the resource url includes the hash delimiter which seems to be causing issues with scene loading in my testing